### PR TITLE
adding the capacity to add multiple secret

### PR DESCRIPTION
**Pull request by waakanni**
onboarding making changes to our charts

### CODE REVIEW

File: charts/auth-service/values.yaml [hunk 1]
```diff
@@ -138,3 +138,5 @@ authService:
 shareProcessNamespace: false
 kafkaSecrets:
   enabled: false
+
+imagePullSecretName: []
```
Comment (ckaipf) [inline charts/auth-service/values.yaml:142]:
```suggestion
imagePullSecretNames: []
```

File: charts/ghga-common/templates/_image-pull-secrets.tpl [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{{- define "ghga-common.image-pull-secrets" -}}
+{{- if .Values.imagePullSecretName -}}
```
Comment (ckaipf) [inline charts/ghga-common/templates/_image-pull-secrets.tpl:2]:
```suggestion
{{- if .Values.imagePullSecretNames -}}
```

File: charts/ghga-common/templates/_image-pull-secrets.tpl [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+imagePullSecrets:
+  {{- toYaml .Values.imagePullSecretName | nindent 6 }}
```
Comment (ckaipf) [inline charts/ghga-common/templates/_image-pull-secrets.tpl:4]:
```suggestion
  {{ toYaml .Values.imagePullSecretNames }}
```

**Comment by ckaipf** [comment]
Thanks, looks good to me. Did you check the following things?

- [ ] Chart is correctly rendered with `helm template`
- [ ] Chart is installable in local or dev Kubernetes cluster
- [ ] Deployment does not crash after installation
- [ ] What happens when secret is set but not available?

Afterwards we could merge it to finalize testing. Then apply to all other charts.

**Comment by waakanni** [comment]
Re: the last 2 checks:

1. Helm deploys the chart without any problem but the pod crashes since it is missing dependants. -> I believe this is the expected behaviour

2. I am not sure how to test this since the pod crashes for reasons. I can't see anything in the logs that mentions this.  I think it will just be ignored. 
@ckaipf

**Review by ckaipf — APPROVED** [review]

**Pull request merged**